### PR TITLE
docs: add accessing SUT from Testkube question

### PR DESCRIPTION
**Pull request by nicufk**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

File: docs/docs/12-FAQ.md [hunk 1]
```diff
@@ -24,4 +24,9 @@ pod/testkube-mongodb-74587998bb-8pzl2                       1/1     Running   0
 pod/testkube-operator-controller-manager-77ffbb8fdc-rxhvx   2/2     Running   0          5d23h
 ```
 
-- If these options do not solve the problem, please contact us on [Discord](https://discord.com/invite/6zupCZFQbe).
\ No newline at end of file
+### How to access the Service Under Test(SUT) using Testkube?
+
+- Services inside the same kubernetes cluster can be accessed using the address \<service-name\>.\<service-namespace\>.svc.cluster.local:\<port-number\>, if there are network restrictions configured, testkube will need permissions to access the SUT over the local network of the cluster
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:29]:
```suggestion
- Services inside the same Kubernetes cluster can be accessed using the address \<service-name\>.\<service-namespace\>.svc.cluster.local:\<port-number\>. If there are network restrictions configured, Testkube will need permissions to access the SUT over the local network of the cluster.
```

File: docs/docs/12-FAQ.md [hunk 2]
```diff
@@ -24,4 +24,9 @@ pod/testkube-mongodb-74587998bb-8pzl2                       1/1     Running   0
...
+- If Testkube and the SUT are not in the same cluster, SUT will have to be exposed to Testkube using an Ingress or a Load Balancer
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:30]:
```suggestion
- If Testkube and the SUT are not in the same cluster, SUT will have to be exposed to Testkube using an Ingress or a Load Balancer.
```

File: docs/docs/12-FAQ.md [hunk 3]
```diff
@@ -24,4 +24,9 @@ pod/testkube-mongodb-74587998bb-8pzl2                       1/1     Running   0
...
+
+If any of these answers does not answer or solve the issue that you encountered, please contact us on [Discord](https://discord.com/invite/6zupCZFQbe).
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:32]:
```suggestion
If this does not solve the issue that you encountered or you have other questions or comments, please contact us on [Discord](https://discord.com/invite/6zupCZFQbe).
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Pull request merged**